src/main/java/echoServer/http/StatusCode.java

```diff
@@ -2,7 +2,8 @@
 
 public enum StatusCode {
     PAGE_NOT_FOUND("HTTP/1.1 404 Not Found\r\nContent-Length: 0\r\n"),
-    BAD_REQUEST("HTTP/1.1 400 Bad Request\r\nContent-Length: 0\r\n");
+    BAD_REQUEST("HTTP/1.1 400 Bad Request\r\nContent-Length: 0\r\n"),
+    OK("HTTP/1.1 200 OK\r\nContent-Length: 0\r\n");
```

Comment:
Ah, I see what StatusCode is now in your code.  I think this is a refactoring opportunity...when googling "http status codes", just the numbers come up (200, 400, 404, etc).  Using these full strings for a `StatusCode` is a bit confusing -- if I'm reading this right, it's really returning the headers and status code in one, right?  These are more Responses than Status Codes, so you may want to separate those two concepts.